lib/buttons_bar.dart

```diff
@@ -0,0 +1,66 @@
+import 'plain_text.dart';
+import 'ui_constants.dart';
+import 'package:flutter/material.dart';
+
+
```

Comment:
remove extra line, if you use the formatting shortcut should be fixed

Reply:
done